### PR TITLE
Adds TOC link to Jira Bridge

### PR DESCRIPTION
**Pull request by manneohrstrom**
To be merged once the jira bridge docs come online.

### CODE REVIEW

File: docs/_data/en/toc_text.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ shotgun: Shotgun
 python-api: Python API
 rest-api: REST API
 event-triggers: Event Driven Triggers
+jira-bridge: Shotgun Jira Bridge (Beta)
 examples: Examples
```
Comment (tannazie) [inline docs/_data/en/toc_text.yml]: Do we want to just call it "Jira Bridge" because it's already in the "Shotgun" section, so it's implied?

Comment (manneohrstrom) [reply docs/_data/en/toc_text.yml]: sure!!! i'll fix it

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/e7da1d3b8338a4b70c3eb96fa5a28af93103a29e/index.html)

**Comment by manneohrstrom** [comment]
To be merged after the jira docs are online.

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/07fd04d1b46f27a7ae6b384925ffa797fa946c6b/index.html)

**Pull request merged**